File: Back/Api/Domain/Event.cs


namespace Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? Date { get; set; }
        public DateTime CreatedAt { get; set; }
        public string Name { get; set; }
        public int QuantityPeople { get; set; }
        public string Lot { get; set; }
        public string ImageURL { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public ICollection<Lot> Lots { get; set; }
        public ICollection<SocialNetwork> SocialNetworks { get; set; }
        public ICollection<EventAssignedSpeaker> EventsAssignedSpeakers { get; set; }
    }
}

File: Front/Events-APP/src/app/models/Event.ts
import { Lot } from "./Lot";
import { SocialNetwork } from "./SocialNetwork";
import { Speaker } from "./Speaker";

export interface Event {
    id: number;
    local: string; 
    date?: Date; 
    createdAt: Date; 
    name: string; 
    quantityPeople: number; 
    imageURL: string; 
    email: string; 
    phone: string; 
    lots: Lot[]; 
    socialNetworks: SocialNetwork[]; 
    speakers: Speaker[]; 
}
